doc/radosgw/cloud-restore.rst

```diff
@@ -1,22 +1,34 @@
-=============
-Cloud Restore
-=============
+===============
+ Cloud Restore 
+===============
 
-`cloud-transition <https://docs.ceph.com/en/latest/radosgw/cloud-transition>`__   feature enables data transition to a remote cloud service. The ``cloud-restore`` feature enables restoration of those transitioned objects from the remote cloud S3 endpoints into RGW.
+The :doc:`cloud-transition>` feature enables transition objects to a remote
```

Comment:
I think that the verb "enables" in "X enables objects to a remote cloud service" isn't quite strong enough to qualify this as a sentence. Is it possible to write this as "The cloud-transition feature makes it possible to transmit data to a remote cloud service."?

I mean, I think I get what cloud-transition is doing. It's facilitating the movement of data objects from a local RGW deployment to a remote S3 store.

Reply:
Editing flern.  I pushed a change.